### PR TITLE
fix(housing): put the travaux tasks in a subpage

### PR DESCRIPTION
**Pull request by euhmeuh**
SCDC-135

### CODE REVIEW

File: client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "WorkingStatusCtrl",
+    class WorkingStatusCtrl {
+        constructor ($scope, Products) {
+            this.$scope = $scope;
+            this.Products = Products;
+            this.worksDetails = [];
+        }
+
+        $onInit () {
+            this.Products.getWorks("dedicated_server", true, true).then((works) => {
```
Comment (JeremyDec) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js]: You could use `({ items })` no ?

File: client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="container-fluid mb-5 px-0">
+<div ui-view class="container-fluid mb-5 px-0">
```
Comment (JeremyDec) [inline client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html]: `data-` ?

File: client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html [hunk 3]
```diff
@@ -269,14 +269,12 @@
...
-                            <div data-working-status
-                                 data-modal-id="'worksDedicatedServers'"
-                                 data-product-type="'dedicated_server'"
-                                 data-works-details="works"
-                                 data-show-affiliated="true"
-                                 data-show-only-active="true"
-                                 data-tr="tr">
-                            </div>
+                            <p data-translate="core_works_description"></p>
+                                <button type="button"
```
Comment (marie-j) [inline client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html]: Why not use `oui-button` ?

Comment (FredericEspiau) [reply client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html]: Also, indentation

File: client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+angular.module("App").controller(
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.controller.js]: You could create a component instead

File: client/app/dedicated/server/working-status/dedicated-server-working-status.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+    <oui-back-button data-ui-sref="app.dedicated.server"
+                     data-title="{{:: 'modal_working_status_title' | translate }}">
+    </oui-back-button>
+
+    <div data-ng-repeat="work in $ctrl.worksDetails track by $index">
+        <h5>
+            <a target="_blank"
+               rel="noopener"
+               title="{{ work.title }} ({{ 'core_new_window' | translate }})"
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.html]: Please do not use `title` attribute: https://www.w3.org/TR/html/dom.html#the-title-attribute

File: client/app/dedicated/server/working-status/dedicated-server-working-status.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+               data-ng-href="{{ work.link }}"
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.html]: `::`

File: client/app/dedicated/server/working-status/dedicated-server-working-status.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+               data-ng-bind="work.title">
+            </a>
+        </h5>
+        <em data-ng-bind="work.date | date:'short'"></em>
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.html]: https://developer.mozilla.org/fr/docs/Web/HTML/Element/time #semantic

File: client/app/dedicated/server/working-status/dedicated-server-working-status.html [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+        <p data-ng-bind-html="work.summary"></p>
+        <ul class="list-inline font-italic">
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.html]: Does it really need to be all italic ?

File: client/app/dedicated/server/working-status/dedicated-server-working-status.html [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+            <li>
+                <strong data-translate="modal_working_status_type"></strong>
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.html]: Shouldn't it be `dt` and `dd` instead?

File: client/app/dedicated/server/working-status/dedicated-server-working-status.routes.js [hunk 10]
```diff
@@ -0,0 +1,7 @@
+angular.module("App").config(($stateProvider) => {
+    $stateProvider.state("app.dedicated.server.working-status", {
+        url: "/travaux",
+        templateUrl: "dedicated/server/working-status/dedicated-server-working-status.html",
+        controller: "WorkingStatusCtrl"
```
Comment (FredericEspiau) [inline client/app/dedicated/server/working-status/dedicated-server-working-status.routes.js]: You could have set the `controllerAs` here, but once again, using a component would be better :)

File: client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html (hunk 3 above)
Comment (euhmeuh) [reply client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html]: Well I tried ussing oui-button but it currently lacks a way to go full-width.
We'll add the ability to do that in oui, but in the mean time, it's a normal button.

**Comment by jleveugle** [comment]
Hello @euhmeuh,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**